pkg/kubelet/config/apiserver.go

```diff
@@ -54,7 +54,7 @@ func NewSourceApiserver(c clientset.Interface, nodeName types.NodeName, nodeHasS
 	}()
 }
 
-// newSourceApiserverFromLW holds creates a config source that watches and pulls from the apiserver.
+// newSourceApiserverFromLW holds a config source that watches and pulls from the apiserver.
```

Comment:
I think the confusion here comes from the naming of the method `new`. New typically means creating something. Consider changing the method name as well to clarify what it does exactly. 

/close

Thank you for your PR. It seems to only contain trivial edits. Please read our [trivial edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits). We encourage you to take a look at confirmed issues and bugs or issues marked `help-wanted`.